src/parser/CMakeLists.txt

```diff
@@ -103,6 +103,15 @@ set_target_properties(cvc4parser PROPERTIES SOVERSION ${CVC4_SOVERSION})
 target_compile_definitions(cvc4parser PRIVATE -D__BUILDING_CVC4PARSERLIB)
 target_link_libraries(cvc4parser PUBLIC cvc4)
 target_link_libraries(cvc4parser PRIVATE ANTLR3)
+
+if(USE_CLN)
+  target_link_libraries(cvc4parser PRIVATE ${CLN_LIBRARIES})
+  target_include_directories(cvc4parser PRIVATE $<BUILD_INTERFACE:${CLN_INCLUDE_DIR}>)
+endif()
+
+target_link_libraries(cvc4parser PRIVATE ${GMP_LIBRARIES})
+target_include_directories(cvc4parser PRIVATE $<BUILD_INTERFACE:${GMP_INCLUDE_DIR}>)
```

Comment:
Shouldn't be needed anymore since Integer/Rational was eliminated in the TPTP parser (https://github.com/CVC4/CVC4/pull/6239).

Reply:
CI seems to disagree: https://github.com/CVC4/CVC4/pull/6218/commits/f0c324ef95e899b73ebf2674577e68820f5b728c / https://github.com/CVC4/CVC4/runs/2231668107